tests/validations/test_unique_valid_name.py

```diff
@@ -33,6 +33,27 @@
 """
 
 
+def test_semantic_model_name_validity(  # noqa: D
+    simple_semantic_manifest__with_primary_transforms: PydanticSemanticManifest,
+):
+    validator = SemanticManifestValidator[PydanticSemanticManifest](
+        [UniqueAndValidNameRule[PydanticSemanticManifest]()]
+    )
+
+    # Shouldn't raise an exception
+    validator.checked_validations(simple_semantic_manifest__with_primary_transforms)
+
+    # Should raise an exception
+    compied_manifest = deepcopy(simple_semantic_manifest__with_primary_transforms)
+    semantic_model = compied_manifest.semantic_models[0]
```

Comment:
nit: `compied` -> `copied`

Reply:
Corrected in: [98e21d0](https://github.com/dbt-labs/dbt-semantic-interfaces/pull/184/commits/98e21d0dd8925475e2cf5f62b7f43cf76265642d)
Will be fixed up into: [bd56473](https://github.com/dbt-labs/dbt-semantic-interfaces/pull/184/commits/bd564730107d965162dab051fcf7e36724408ac7)